catalog/plugin/catalog-plugin-videothumbnail/src/main/java/org/codice/ddf/catalog/content/plugin/video/VideoThumbnailPlugin.java

```diff
@@ -175,27 +184,29 @@ private boolean isVideo(final ContentItem contentItem) {
     private void createThumbnail(final ContentItem contentItem) throws PluginExecutionException {
         LOGGER.debug("About to create video thumbnail.");
 
+        Path contentPath = Paths.get(tempDirectory.toAbsolutePath()
+                .toString(), contentItem.getFilename());
         try {
+
+            Files.copy(contentItem.getInputStream(), contentPath);
```

Comment:
This is going to kill performance. You already had it saved off to disk before, now you have to copy it to a new location and then read it back in again. I say just leave it the way it was.